### PR TITLE
Update help docs

### LINKED ISSUE #2225: Add to FAQ: why no way to upload file manually
Feedback from PyPI form: "Manual Upload URL not working"

> is it me or there's no way to upload packages manually ? this transition kind of sucks because the given link to do manual upload doesn't work (https://upload.pypi.org/legacy/)

### LINKED ISSUE #2659: Update "Common Questions" regarding maintenance
`warehouse/templates/pages/help.html` (the source for https://pypi.org/help/ ) has a few sections that are ready for improvement:

* What does pre-production mean?
* Who maintains PyPI?
* How can I help?

We should point to [our milestones](https://github.com/pypa/warehouse/milestones), our [good first issue label for our issues](https://github.com/pypa/warehouse/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22), and point to relevant mailing lists where people can keep updated on upcoming changes (such as [distutils-sig](https://mail.python.org/mailman/listinfo/distutils-sig) and [pypa-dev](https://groups.google.com/forum/#!forum/pypa-dev)).

### PR DESCRIPTION
**Pull request by nlhkabu**
Closes #2659, Closes #2225 

screencapture-localhost-help-1513705521862

### CODE REVIEW

**Comment by ewdurbin** [comment]
Ohhhhh while we're in here @nlhkabu can we add a clickable/copyable anchor link for each `id`?

**Comment by nlhkabu** [comment]
@ewdurbin already done :grin:

File: warehouse/templates/pages/help.html [hunk 1]
```diff
@@ -36,23 +65,57 @@ <h3 id="packages" class="common-question">What's a package, project, or release?
...
+
+      <p>'Pre-production' indicates that you are using a version of this software that is not fully tested and 'production ready'. This means:
+      <ul>
+        <li>The UI may return unusual or erroneous results</li>
+        <li>Performance may not yet be optimized</li>
+        <li>Some features that you use on the old interface may be missing</li>
+      </ul>
+
+      <p>While we are in pre-production, our infrastructure cannot (yet!) support all of the traffic generated by our users running <code>pip install</code>. We therefore ask that you do not use this version of PyPI for production installations.</p>
+      <p>On the other hand, uploads are working well and we encourage project maintainers and owners to use this version of PyPI over the legacy site.</p>
```
Comment (dstufft) [inline warehouse/templates/pages/help.html]: One slight tweak here, not sure if it matters but more than encouraging people to upload using Warehouse, we *require* it. Uploading to legacy no longer functions. Not sure if it makes sense to change this wording or not.

Comment (nlhkabu) [reply warehouse/templates/pages/help.html]: Proposed new text:

`On the other hand, uploads are working well and project maintainers and owners should use this version of PyPI over the legacy site, which no longer supports this function.`

is this ok @dstufft ?

Comment (dstufft) [reply warehouse/templates/pages/help.html]: Yup, sounds good.

**Comment by nlhkabu** [comment]
Cleaned up PR commit history, and updated text based on feedback:

screencapture-localhost-help-1513705521862

**Comment by nlhkabu** [comment]
ping @dstufft for final review.  If you're happy I'll rebase and merge :)

**Pull request merged**